<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingwindow</class>
 <widget class="QWidget" name="settingwindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="d3cam.qrc">
    <normaloff>:/D3Cam/Win32/Debug/setting .ico</normaloff>:/D3Cam/Win32/Debug/setting .ico</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">
QRadioButton:indicator{
    width:17px; 
    height:17px;
}
QRadioButton::indicator::unchecked{
	image: url(:/D3Cam/Win32/Debug/radiobutton-unchecked-.png);
}
QRadioButton::indicator::checked{
	image: url(:/D3Cam/Win32/Debug/radiobutton-checked.png);
}
QPushButton{
	background-color: rgba(50, 90, 100, 100);
    border-style:dash;
    border-width:5px;
    border-radius:9px;
	border-color: rgba(255, 255, 255, 30);
	font: 12pt;
}
QPushButton:hover{
	background-color: rgba(50, 190, 255, 100);
	color: rgba(0, 0, 0, 200);
	border-color: rgba(255, 255, 255, 30);
}
QPushButton:pressed{
	background-color: rgba(100, 255, 100, 200);
	border-color: rgba(255, 255, 255, 0);
    border-style:inset;
	color: rgba(0, 0, 0,100);
}
QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px;
    width: 10px;
    margin: -3px 0 -3px 0;
    border-radius: 5px;
}

QSlider:sub-page:horizontal{
    height:1px;
	border-radius:3px;
	
	border-image: url(:/D3Cam/Win32/Debug/无标题1.png);
    
}
QSlider::groove:horizontal {
    subcontrol-origin: content;
    background: rgb(200, 200, 200);
    height: 10px;
}
QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    selection-background-color: darkgray;
}

QComboBox {
    border: 1px solid gray;
    border-radius: 5px;
    padding: 1px 2px 1px 2px;
}

QComboBox QAbstractItemView {
    border: 2px solid darkgray;
    selection-background-color: lightgray;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
 
    border-left-width: 2px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow{
	border-image: url(:/D3Cam/Win32/Debug/arrow_down.png);
    width:20px;
}

QMainWindow{
	border-radius:5px;
	border-color: rgb(85, 170, 255);
    
	background-color: rgb(255, 255, 0);
}

QGroupBox{
border-radius:5px;
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>30</y>
     <width>521</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>相机设置</string>
    </attribute>
    <attribute name="toolTip">
     <string>设置相机的白平衡、曝光与增益</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>自动白平衡：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBtnAutoWhiteOpen">
         <property name="text">
          <string>是</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBtnAutoWhiteClose">
         <property name="text">
          <string>否</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnResetWhiteBalance">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>重置</string>
         </property>
         <property name="icon">
          <iconset resource="d3cam.qrc">
           <normaloff>:/D3Cam/Win32/Debug/reset.ico</normaloff>:/D3Cam/Win32/Debug/reset.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>19</width>
           <height>19</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>曝光：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderExp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="editExp">
       <property name="font">
        <font>
         <family>Malgun Gothic</family>
         <pointsize>9</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;Malgun Gothic&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>增益：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderGain">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="editGain">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;Malgun Gothic&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>定时器设置</string>
    </attribute>
    <attribute name="toolTip">
     <string>设置定时器工作时间和间隔时间</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="portnamecomboBox">
         <property name="font">
          <font>
           <family>Malgun Gothic</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Malgun Gothic&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>COM1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM9</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openBtn">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>打开串口</string>
         </property>
         <property name="icon">
          <iconset resource="d3cam.qrc">
           <normaloff>:/D3Cam/Win32/Debug/serialport1.ico</normaloff>:/D3Cam/Win32/Debug/serialport1.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="downBtn">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>关闭串口</string>
         </property>
         <property name="icon">
          <iconset resource="d3cam.qrc">
           <normaloff>:/D3Cam/Win32/Debug/serialport2.net.ico</normaloff>:/D3Cam/Win32/Debug/serialport2.net.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>间隔时间：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QTextBrowser" name="textBrowser_send1">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;w&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>工作时间：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="sendBtn1">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;宋体&quot;;
background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="sendBtn2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTextBrowser" name="textBrowser_send2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;s&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QTextBrowser" name="textBrowser_receive">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>接收区：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>图像下载与上传</string>
    </attribute>
    <attribute name="toolTip">
     <string>从相机存储中下载图像、上传触发文件至相机中</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>11</y>
          <width>72</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>下载到：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="editImgDownloadDir">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Malgun Gothic&quot;;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="btnChooseImgDownloadDir">
        <property name="geometry">
         <rect>
          <x>373</x>
          <y>10</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;宋体&quot;;
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="downloadImgBtn">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>54</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;宋体&quot;;
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>下载</string>
        </property>
        <property name="icon">
         <iconset resource="d3cam.qrc">
          <normaloff>:/D3Cam/Win32/Debug/download.ico</normaloff>:/D3Cam/Win32/Debug/download.ico</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDeleteImgs">
        <property name="geometry">
         <rect>
          <x>123</x>
          <y>54</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);
font: 11pt &quot;宋体&quot;;
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset resource="d3cam.qrc">
          <normaloff>:/D3Cam/Win32/Debug/delete.ico</normaloff>:/D3Cam/Win32/Debug/delete.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>触发模式选择</string>
       </property>
       <widget class="QPushButton" name="uploadProgramBtn">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;宋体&quot;;
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>选择触发文件并上传....</string>
        </property>
        <property name="icon">
         <iconset resource="d3cam.qrc">
          <normaloff>:/D3Cam/Win32/Debug/folder_open.ico</normaloff>:/D3Cam/Win32/Debug/folder_open.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);
border-radius:3px;
font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>                          设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>-9</x>
     <y>324</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);
border-radius:3px;
font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>251</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="rightMargin">
     <number>7</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="btnSaveCamConfig">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>保存设置</string>
      </property>
      <property name="icon">
       <iconset resource="d3cam.qrc">
        <normaloff>:/D3Cam/Win32/Debug/save.png</normaloff>:/D3Cam/Win32/Debug/save.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnShutDownCam">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>断电</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 0);

font: 11pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>关机</string>
      </property>
      <property name="icon">
       <iconset resource="d3cam.qrc">
        <normaloff>:/D3Cam/Win32/Debug/shutdown.ico</normaloff>:/D3Cam/Win32/Debug/shutdown.ico</iconset>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>btnShutDownCam</zorder>
   <zorder>btnSaveCamConfig</zorder>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QToolButton" name="minButton">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="closeButton">
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>tabWidget</zorder>
  <zorder>label_8</zorder>
  <zorder>splitter</zorder>
  <zorder>label_9</zorder>
 </widget>
 <resources>
  <include location="d3cam.qrc"/>
 </resources>
 <connections/>
</ui>
